src/lib.rs

```diff
@@ -51,6 +51,10 @@ pub mod calendars;
 #[macro_use]
 pub mod objects;
 mod enhancers;
+#[cfg(not(feature = "parser"))]
+pub(crate) mod file_handler; // to keep backward compatibility, not exposing FileHandler
```

Comment:
You probably can remove the comment, I did put a comment in the snippet I've sent on private message to explain why I was doing it.

Reply:
I thought it was interesting, but I’m going to remove it